### ISSUE TITLE
Missing audio description for the video

### ISSUE THREAD

**Issue by jrbenn** [strategicdata/pmhc-mds-user-documentation#13]
Audio description is missing for the video. In the absence of audio description, visual impairment users could not understand the important visual content conveyed in the video.

Issue #232 in https://docs.google.com/spreadsheets/d/13iTb4aW7KulDcbj5Q2xD78H4KOrMmhvfIDW1JMffY7o/edit?gid=155984173#gid=155984173
